adyenv6core/src/com/adyen/v6/service/DefaultAdyenOrderService.java

```diff
@@ -148,6 +148,9 @@ public void updateOrderFromPaymentsResponse(OrderModel order, PaymentsResponse p
         paymentInfo.setAdyenMultibancoDeadline(paymentsResponse.getMultibancoDeadline());
         paymentInfo.setAdyenMultibancoReference(paymentsResponse.getMultibancoReference());
 
+        //pos receipt
+        paymentInfo.setAdyenPosReceipt(paymentsResponse.getAdditionalDataByKey("pos.receipt"));
```

Comment:
remove empty line

Reply:
It is intentional for formatting.

Reply:
this is kept intentional for formatting.